MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -60,7 +91,8 @@ public string VideoRange
 
                 var colorTransfer = ColorTransfer;
 
-                if (string.Equals(colorTransfer, "smpte2084", StringComparison.OrdinalIgnoreCase))
+                if (string.Equals(colorTransfer, "smpte2084", StringComparison.OrdinalIgnoreCase)
+                || string.Equals(colorTransfer, "arib-std-b67", StringComparison.OrdinalIgnoreCase))
                 {
```

Comment:
`It may not be clear from this description but VS10 supports HDR10. The playback solution for devices will receive any HDR format based on SMPTE ST-2084 and ST-2086. Of course, it delivers the best HDR experience with Dolby Vision. Devices based on Dolby Vision VS10 will play back a variety of HDR content types, including HDR10 from Ultra HD Blu-ray and UHDA-certified HDR content. `

` This includes ST 2094-10 (Dolby Vision format), Colour Volume Reconstruction Information (CVRI) SMPTE ST 2094-20 (Philips format) and Colour Remapping Information (CRI) defined in ST 2094-30 (Technicolor format), and HDR10+ ST 2094-40 (Samsung format).`

https://en.m.wikipedia.org/wiki/High-dynamic-range_video

Found a few more. Should probably find proper samples to feed ffmpeg